src/EditorFeatures/Core/Implementation/CodeFixes/CodeFixService.cs

```diff
@@ -404,9 +431,9 @@ await diagnosticsWithSameSpan.OrderByDescending(d => d.Severity)
             }
 
             // If the fix provider supports fix all occurrences, then get the corresponding FixAllProviderInfo and fix all context.
-            var fixAllProviderInfo = extensionManager.PerformFunction(fixer, () => ImmutableInterlocked.GetOrAdd(ref _fixAllProviderMap, fixer, FixAllProviderInfo.Create), defaultValue: null);
+            var fixAllProviderInfo = extensionManager.PerformFunction(fixer, () => ImmutableInterlocked.GetOrAdd(ref _fixAllProviderMap, fixer, FixAllProviderInfo.Create), defaultValue: null!);
```

Comment:
❕ This suppression is not correct. The generic type argument for `PerformFunction` should be specified explicitly since type inference produced an incorrect result for this call.